tests/cli/drive-scanner.spec.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const m = require('mochainon');
+const drivelist = require('drivelist');
+
+describe('CLI: DriveScanner', function() {
+
+  describe('given no available drives', function() {
+
+    beforeEach(function() {
+      this.scanner = require('../../lib/cli/drive-scanner');
```

Comment:
Yes, it doesn't help with anything since it's a singleton anyway.
This brings up a concern. Do tests run in parallel? If I have a test that needs the scanner to not be started to function correctly, and another test starts the scanner, since the scanner is a singleton, will the tests interfere?